R/library.R

```diff
@@ -42,13 +54,24 @@ install_package <- function(locked_package, libPath, quiet) {
   UseMethod("install_package")
 }
 
+load_package <- function(locked_package) {
+  UseMethod("load_package")
+}
+
 install_package.local <- function(locked_package, libPath, quiet) {
   stopifnot(is.element("dir", names(locked_package)))
 
   utils::install.packages(locked_package$dir, lib = libPath, repos = NULL
     , type = "source", quiet = quiet)
 }
 
+load_package.local <- function(locked_package) {
+  stopifnot(is.element("dir", names(locked_package)))
+  cat("Reinstalling", crayon_green(locked_package$name),
+    as.character(locked_package$version), "from", class(locked_package)[1], "\n")
```

Comment:
Maybe `if (!requireNamespace("devtools", quietly = TRUE))` stop and warn the user they must install devtools first.